### ISSUE TITLE
A training question about GENet

### ISSUE THREAD

**Issue by FisherWY** [idstcv/ZenNAS#29]
Hi, everyone. I'm currently trying to reproduce your previous work "Neural Architecture Design for GPU-Efficient Networks", here is the repo [link](https://github.com/idstcv/GPU-Efficient-Networks). After determining the model structure, such as "GENet_large", "GENet_small", etc. Can we refer to the training script([train_image_classification.py](https://github.com/idstcv/ZenNAS/blob/main/train_image_classification.py)) in this repo to train the model and get the effect consistent with the paper description?

**Comment by MingLin-home** [comment]
Hi FisherWY,

Sorry for the late reply. We no longer maintain GENet repo. Yes you can use Zen-NAS training script. Actually you can use the more popular TIMM framework to train our model. Teacher-Student Distillation is strongly recommended for better accuracy. The TS training script is here:

https://github.com/idstcv/ZenNAS/blob/main/ts_train_image_classification.py

**Comment by FisherWY** [comment]
Thank you for your guidance, I will try it according to your suggestion. :-)
